src/setupTests.js

```diff
@@ -1 +1 @@
-import '@testing-library/jest-dom';
\ No newline at end of file
+import '@testing-library/jest-dom';
```

Comment:
Well done writing the integration tests using React Testing Library for some pure functions in Redux code 👏🏾 💯 

- Kindly write unit tests for the several components that in the components directory.